src/layers/SelectionLayer.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable no-underscore-dangle */
+// File adopted from nebula.gl's SelectionLayer
+// https://github.com/uber/nebula.gl/blob/master/modules/layers/src/layers/selection-layer.js
```

Comment:
Slight preference for pointing to a versioned or tagged URL: this one is more likely to change over time.

Reply:
That makes sense, I will update.